LiveLessons4/Lesson4_4_generics.playground/contents.swift

```diff
@@ -35,7 +35,8 @@ func fmap<T, U>(value: T?, f: T -> U) -> U? {
     }
 }
 var greeting: String? = "Bonjour!"
-let charCount = fmap(greeting) { count($0) }
+
+let charCount = fmap(greeting) { $0.characters.count } 
```

Comment:
I must say I'm confused . If fmap() would be a normal function and not a generic I would totally understand "why". But it's a generic so "how" does it now that it's a String instance? :)